tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -898,7 +897,7 @@ def eltwise_bitwise_not(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttnn.bitwise_not(t0, value, memory_config=output_mem_config, queue_id=0)
+    t1 = ttnn.bitwise_not(t0, memory_config=output_mem_config, queue_id=0)
```

Comment:
Besides this, is there any test that need to update?  BTW, how do I run tests locally?